nibabel/streamlines/trk.py

```diff
@@ -101,9 +101,12 @@ def get_affine_trackvis_to_rasmm(header):
     # If the voxel order implied by the affine does not match the voxel
     # order in the TRK header, change the orientation.
     # voxel (header) -> voxel (affine)
-    header_ornt = asstr(header[Field.VOXEL_ORDER])
+    vox_order = header[Field.VOXEL_ORDER]
+    # Input header can be dict or structured array
+    if hasattr(vox_order, 'item'):  # structured array
+        vox_order = header[Field.VOXEL_ORDER].item()
```

Comment:
I believe we need a string as input to `axcodes2ornt`, rather than a byte string.  Thence we need to decode the byte string in `header[Field.VOXEL_ORDER]`.  The numpy object doesn't have the `decode` method, so I think we have to fetch the underlying bytes object in order to decode it (this all for Python 3).